op-program/host/flags/flags.go

```diff
@@ -184,5 +191,8 @@ func CheckRequired(ctx *cli.Context) error {
 			return fmt.Errorf("flag %s is required", flag.Names()[0])
 		}
 	}
+	if !ctx.IsSet(L2OutputRoot.Name) && !ctx.IsSet(L2AgreedPrestate.Name) {
+		return fmt.Errorf("flag %s or %s is required", L2OutputRoot.Name, L2AgreedPrestate.Name)
+	}
```

Comment:
We should also prevent both flags from being set. To prevent inconsistent prestate and roots flags from being specified.

Reply:
Done.